CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [[PR #434](https://github.com/nf-core/chipseq/pull/434)] - Prevent pipeline fails from erroneous param validation when igenomes is used.
 - [[#432](https://github.com/nf-core/chipseq/issues/432)] - Fix `GFFREAD` call to have the two expected input channels.
+- [[#428](https://github.com/nf-core/chipseq/issues/428)] - Bump MultiQC version to 1.25.1
```

Comment:
Please update changelog with the corresponding message, this refers to the previous PR #436